### ISSUE TITLE
Downloading SEC Form 4 Error

### ISSUE THREAD

**Issue by Darkersideofthemoon** [jadchaar/sec-edgar-downloader#121]
I have used the same script in Python to download SEC form 4 for a long time and it no longer works.  In fact, I cannot download any SEC forms using SEC Edgar downloader/

**Comment by PatrickOster1993** [comment]
I have the same issue. Seems that the form 4 cannot be downloaded anymore - no matter which ticker I choose.

**Comment by jadchaar** [comment]
Hi all, this should now be fixed in v5.0, which is a full re-write of the package. I have just confirmed form 4 downloads work as expected in v5:

```
>>> from sec_edgar_downloader import Downloader
>>> dl = Downloader("Personal", "foo@bar.com")
>>> dl.get("4", "AAPL", limit=5)
5
```
